DrewBradley/ticTacToe-2010fe-solo-project
# Check HTML in WAVE. Add media queries to CSS to optomize for mobile.

#### What does this PR do?
Updates the HTML to be more semantic for accessibility. Removes any superfluous CSS, and adds media queries to optimize the site for mobile phones. Site was run through WAVE (both desktop and mobile) and returned 9 errors, which correspond with the tic tac toe squares. This is due to them being empty buttons at the top of the game.

#### Where should the reviewer start?
Review media queries at the bottom of the styles.css file. Please look over HTML and js files for any inconsistencies.

#### How should this PR be manually tested?
Launch ```index.html``` on your browser. If you are using Chrome, you can double-check the mobile layout by open Dev Tools and toggling between desktop and mobile view.

#### Any additional considerations?
Please review media queries closely to determine validity. Please keep in mind that CSS styles are inherited, even after media queries, so some styles must be rewritten to prevent it from defaulting to original styles.
